packages/xo-server/src/api/host.js

```diff
@@ -215,6 +215,20 @@ emergencyShutdownHost.resolve = {
 
 // -------------------------------------------------------------------
 
+export async function assertConsistentHostServerTime({ host }) {
+  return this.getXapi(host)._assertConsistentHostServerTime(host._xapiRef)
```

Comment:
`_assertConsistentHostServerTime` is a private method which cannot be call out side its class. Please make it public.

Reply:
I don't see why I change it to public method? it reserved for host.

Reply:
A private method should not be called by the instance.